### PR TITLE
Update network debug troubleshooting guide

### LINKED ISSUE #35914: vyos_command module throwing socket_path does not exist or cannot be found
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
vyos_command

##### ANSIBLE VERSION
```
ansible 2.5.0
  config file = /home/vagrant/.ansible.cfg
  configured module search path = [u'/home/vagrant/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, Aug  4 2017, 00:39:18) [GCC 4.8.5 20150623 (Red Hat 4.8.5-16)]
```

##### CONFIGURATION
```
[vagrant@ansible test]$ ansible-config dump --only-changed
DEFAULT_GATHERING(/home/vagrant/.ansible.cfg) = explicit
DEFAULT_HOST_LIST(/home/vagrant/.ansible.cfg) = [u'/home/vagrant/ansible-networking-vagrant-demo/training-course/hosts']
DEPRECATION_WARNINGS(/home/vagrant/.ansible.cfg) = False
HOST_KEY_CHECKING(/home/vagrant/.ansible.cfg) = False
PARAMIKO_HOST_KEY_AUTO_ADD(/home/vagrant/.ansible.cfg) = True
```

##### OS / ENVIRONMENT
CentOS Linux release 7.4.1708 (Core)

##### SUMMARY
vyos_command module is throwing an error `socket_path does not exist or cannot be found` only when the command being run will return a failure.

##### STEPS TO REPRODUCE
```yaml
---
- hosts: network
  connection: network_cli
  tasks:
    - name: Test reachability spine01 to leafs
      vyos_command:
        commands:
          - "ping 192.168.11.2 count 2"
          - "ping 192.168.12.2 count 2"
      register: spine01_results
      when: 'inventory_hostname == "spine01"'

    - name: Test reachability spine02 to leafs
      vyos_command:
        commands:
          - "ping 192.168.21.4 count 2"
          - "ping 192.168.22.4 count 2"
      register: spine02_results
      when: 'inventory_hostname == "spine02"'
```
The 192.168.11.2 and 192.168.12.2 for spine01 pings do work and exist.  For the 2nd task the ping 192.168.21.4 and 192.168.22.4 are wrong, it should fail.  On the command line it works fine->
```
vagrant@spine02:~$ ping 192.168.22.4 count 2
PING 192.168.22.4 (192.168.22.4) 56(84) bytes of data.

--- 192.168.22.4 ping statistics ---
2 packets transmitted, 0 received, 100% packet loss, time 1002ms
```

##### EXPECTED RESULTS
something like this, basically the task should complete, but the error can be registered.  

```yaml
TASK [Test reachability spine01 to leafs] *******************************************************************************************
task path: /home/vagrant/ansible-networking-vagrant-demo/test/check.yml:5
skipping: [leaf01] => {
    "changed": false,
    "skip_reason": "Conditional result was False",
    "skipped": true
}
skipping: [leaf02] => {
    "changed": false,
    "skip_reason": "Conditional result was False",
    "skipped": true
}
skipping: [spine02] => {
    "changed": false,
    "skip_reason": "Conditional result was False",
    "skipped": true
}
<172.16.10.13> attempting to start connection
<172.16.10.13> using connection plugin network_cli
<172.16.10.13> local domain socket does not exist, starting it
<172.16.10.13> control socket path is /home/vagrant/.ansible/pc/b3051e31cc
<172.16.10.13> connection to remote device started successfully
<172.16.10.13> local domain socket listeners started successfully
<172.16.10.13>
<172.16.10.13> local domain socket path is /home/vagrant/.ansible/pc/b3051e31cc
Using module file /usr/lib/python2.7/site-packages/ansible/modules/network/vyos/vyos_command.py
<172.16.10.13> ESTABLISH LOCAL CONNECTION FOR USER: vagrant
<172.16.10.13> EXEC /bin/sh -c 'echo ~ && sleep 0'
<172.16.10.13> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/vagrant/.ansible/tmp/ansible-tmp-1518108133.18-277550148407321 `" && echo ansible-tmp-1518108133.18-277550148407321="` echo /home/vagrant/.ansible/tmp/ansible-tmp-1518108133.18-277550148407321 `" ) && sleep 0'
<172.16.10.13> PUT /home/vagrant/.ansible/tmp/ansible-local-8825rmnY9F/tmpGUg4oA TO /home/vagrant/.ansible/tmp/ansible-tmp-1518108133.18-277550148407321/vyos_command.py
<172.16.10.13> EXEC /bin/sh -c 'chmod u+x /home/vagrant/.ansible/tmp/ansible-tmp-1518108133.18-277550148407321/ /home/vagrant/.ansible/tmp/ansible-tmp-1518108133.18-277550148407321/vyos_command.py && sleep 0'
<172.16.10.13> EXEC /bin/sh -c '/usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1518108133.18-277550148407321/vyos_command.py && sleep 0'
ok: [spine01] => {
    "changed": false,
    "failed": false,
    "invocation": {
        "module_args": {
            "commands": [
                "ping 192.168.11.2 count 2",
                "ping 192.168.12.2 count 2"
            ],
            "host": null,
            "interval": 1,
            "match": "all",
            "password": null,
            "port": null,
            "provider": null,
            "retries": 10,
            "ssh_keyfile": null,
            "timeout": null,
            "username": null,
            "wait_for": null
        }
    },
    "stdout": [
        "PING 192.168.11.2 (192.168.11.2) 56(84) bytes of data.\n64 bytes from 192.168.11.2: icmp_req=1 ttl=64 time=0.383 ms\n64 bytes from 192.168.11.2: icmp_req=2 ttl=64 time=0.285 ms\n\n--- 192.168.11.2 ping statistics ---\n2 packets transmitted, 2 received, 0% packet loss, time 999ms\nrtt min/avg/max/mdev = 0.285/0.334/0.383/0.049 ms",
        "PING 192.168.12.2 (192.168.12.2) 56(84) bytes of data.\n64 bytes from 192.168.12.2: icmp_req=1 ttl=64 time=0.418 ms\n64 bytes from 192.168.12.2: icmp_req=2 ttl=64 time=0.383 ms\n\n--- 192.168.12.2 ping statistics ---\n2 packets transmitted, 2 received, 0% packet loss, time 1000ms\nrtt min/avg/max/mdev = 0.383/0.400/0.418/0.026 ms"
    ],
    "stdout_lines": [
        [
            "PING 192.168.11.2 (192.168.11.2) 56(84) bytes of data.",
            "64 bytes from 192.168.11.2: icmp_req=1 ttl=64 time=0.383 ms",
            "64 bytes from 192.168.11.2: icmp_req=2 ttl=64 time=0.285 ms",
            "",
            "--- 192.168.11.2 ping statistics ---",
            "2 packets transmitted, 2 received, 0% packet loss, time 999ms",
            "rtt min/avg/max/mdev = 0.285/0.334/0.383/0.049 ms"
        ],
        [
            "PING 192.168.12.2 (192.168.12.2) 56(84) bytes of data.",
            "64 bytes from 192.168.12.2: icmp_req=1 ttl=64 time=0.418 ms",
            "64 bytes from 192.168.12.2: icmp_req=2 ttl=64 time=0.383 ms",
            "",
            "--- 192.168.12.2 ping statistics ---",
            "2 packets transmitted, 2 received, 0% packet loss, time 1000ms",
            "rtt min/avg/max/mdev = 0.383/0.400/0.418/0.026 ms"
        ]
    ]
}

TASK [Test reachability spine02 to leafs] *******************************************************************************************
task path: /home/vagrant/ansible-networking-vagrant-demo/test/check.yml:5
skipping: [leaf01] => {
    "changed": false,
    "skip_reason": "Conditional result was False",
    "skipped": true
}
skipping: [leaf02] => {
    "changed": false,
    "skip_reason": "Conditional result was False",
    "skipped": true
}
skipping: [spine02] => {
    "changed": false,
    "skip_reason": "Conditional result was False",
    "skipped": true
}
<172.16.10.13> attempting to start connection
<172.16.10.13> using connection plugin network_cli
<172.16.10.13> local domain socket does not exist, starting it
<172.16.10.13> control socket path is /home/vagrant/.ansible/pc/b3051e31cc
<172.16.10.13> connection to remote device started successfully
<172.16.10.13> local domain socket listeners started successfully
<172.16.10.13>
<172.16.10.13> local domain socket path is /home/vagrant/.ansible/pc/b3051e31cc
Using module file /usr/lib/python2.7/site-packages/ansible/modules/network/vyos/vyos_command.py
<172.16.10.13> ESTABLISH LOCAL CONNECTION FOR USER: vagrant
<172.16.10.13> EXEC /bin/sh -c 'echo ~ && sleep 0'
<172.16.10.13> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/vagrant/.ansible/tmp/ansible-tmp-1518108133.18-277550148407321 `" && echo ansible-tmp-1518108133.18-277550148407321="` echo /home/vagrant/.ansible/tmp/ansible-tmp-1518108133.18-277550148407321 `" ) && sleep 0'
<172.16.10.13> PUT /home/vagrant/.ansible/tmp/ansible-local-8825rmnY9F/tmpGUg4oA TO /home/vagrant/.ansible/tmp/ansible-tmp-1518108133.18-277550148407321/vyos_command.py
<172.16.10.13> EXEC /bin/sh -c 'chmod u+x /home/vagrant/.ansible/tmp/ansible-tmp-1518108133.18-277550148407321/ /home/vagrant/.ansible/tmp/ansible-tmp-1518108133.18-277550148407321/vyos_command.py && sleep 0'
<172.16.10.13> EXEC /bin/sh -c '/usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1518108133.18-277550148407321/vyos_command.py && sleep 0'
ok: [spine01] => {
    "changed": false,
    "failed": false,
    "invocation": {
        "module_args": {
            "commands": [
                "ping 192.168.21.2 count 2",
                "ping 192.168.22.2 count 2"
            ],
            "host": null,
            "interval": 1,
            "match": "all",
            "password": null,
            "port": null,
            "provider": null,
            "retries": 10,
            "ssh_keyfile": null,
            "timeout": null,
            "username": null,
            "wait_for": null
        }
    },
    "stdout": [
        "PING 192.168.11.2 (192.168.21.4) 56(84) bytes of data.\n64 bytes from 192.168.21.1: icmp_req=1 ttl=64 time=0.383 ms\n64 bytes from 192.168.11.2: icmp_req=2 ttl=64 time=0.285 ms\n\n--- 192.168.11.2 ping statistics ---\n2 packets transmitted, 0 received, 100% packet loss, time 999ms\nrtt min/avg/max/mdev = 0.285/0.334/0.383/0.049 ms",
        "PING 192.168.22.4 (192.168.22.4) 56(84) bytes of data.\n64 bytes from 192.168.22.1: icmp_req=1 ttl=64 time=0.418 ms\n64 bytes from 192.168.12.2: icmp_req=2 ttl=64 time=0.383 ms\n\n--- 192.168.12.2 ping statistics ---\n2 packets transmitted, 0 received, 100% packet loss, time 1000ms\nrtt min/avg/max/mdev = 0.383/0.400/0.418/0.026 ms"
    ],
    "stdout_lines": [
        [
            "PING 192.168.11.2 (192.168.11.2) 56(84) bytes of data.",
            "",
            "--- 192.168.11.2 ping statistics ---",
            "2 packets transmitted, 0 received, 100% packet loss, time 999ms",
            "rtt min/avg/max/mdev = 0.285/0.334/0.383/0.049 ms"
        ],
        [
            "PING 192.168.22.4 (192.168.22.4) 56(84) bytes of data.",
            "                       ",
            "--- 192.168.22.4 ping statistics ---",
            "",
            "--- 192.168.22.4 ping statistics ---",
            "2 packets transmitted, 0 received, 100% packet loss, time 1002ms",
        ]
    ]
}
```

##### ACTUAL RESULTS
```
TASK [Test reachability spine01 to leafs] *******************************************************************************************
task path: /home/vagrant/ansible-networking-vagrant-demo/test/check.yml:5
skipping: [leaf01] => {
    "changed": false,
    "skip_reason": "Conditional result was False",
    "skipped": true
}
skipping: [leaf02] => {
    "changed": false,
    "skip_reason": "Conditional result was False",
    "skipped": true
}
skipping: [spine02] => {
    "changed": false,
    "skip_reason": "Conditional result was False",
    "skipped": true
}
<172.16.10.13> attempting to start connection
<172.16.10.13> using connection plugin network_cli
<172.16.10.13> local domain socket does not exist, starting it
<172.16.10.13> control socket path is /home/vagrant/.ansible/pc/b3051e31cc
<172.16.10.13> connection to remote device started successfully
<172.16.10.13> local domain socket listeners started successfully
<172.16.10.13>
<172.16.10.13> local domain socket path is /home/vagrant/.ansible/pc/b3051e31cc
Using module file /usr/lib/python2.7/site-packages/ansible/modules/network/vyos/vyos_command.py
<172.16.10.13> ESTABLISH LOCAL CONNECTION FOR USER: vagrant
<172.16.10.13> EXEC /bin/sh -c 'echo ~ && sleep 0'
<172.16.10.13> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/vagrant/.ansible/tmp/ansible-tmp-1518108133.18-277550148407321 `" && echo ansible-tmp-1518108133.18-277550148407321="` echo /home/vagrant/.ansible/tmp/ansible-tmp-1518108133.18-277550148407321 `" ) && sleep 0'
<172.16.10.13> PUT /home/vagrant/.ansible/tmp/ansible-local-8825rmnY9F/tmpGUg4oA TO /home/vagrant/.ansible/tmp/ansible-tmp-1518108133.18-277550148407321/vyos_command.py
<172.16.10.13> EXEC /bin/sh -c 'chmod u+x /home/vagrant/.ansible/tmp/ansible-tmp-1518108133.18-277550148407321/ /home/vagrant/.ansible/tmp/ansible-tmp-1518108133.18-277550148407321/vyos_command.py && sleep 0'
<172.16.10.13> EXEC /bin/sh -c '/usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1518108133.18-277550148407321/vyos_command.py && sleep 0'
ok: [spine01] => {
    "changed": false,
    "failed": false,
    "invocation": {
        "module_args": {
            "commands": [
                "ping 192.168.11.2 count 2",
                "ping 192.168.12.2 count 2"
            ],
            "host": null,
            "interval": 1,
            "match": "all",
            "password": null,
            "port": null,
            "provider": null,
            "retries": 10,
            "ssh_keyfile": null,
            "timeout": null,
            "username": null,
            "wait_for": null
        }
    },
    "stdout": [
        "PING 192.168.11.2 (192.168.11.2) 56(84) bytes of data.\n64 bytes from 192.168.11.2: icmp_req=1 ttl=64 time=0.383 ms\n64 bytes from 192.168.11.2: icmp_req=2 ttl=64 time=0.285 ms\n\n--- 192.168.11.2 ping statistics ---\n2 packets transmitted, 2 received, 0% packet loss, time 999ms\nrtt min/avg/max/mdev = 0.285/0.334/0.383/0.049 ms",
        "PING 192.168.12.2 (192.168.12.2) 56(84) bytes of data.\n64 bytes from 192.168.12.2: icmp_req=1 ttl=64 time=0.418 ms\n64 bytes from 192.168.12.2: icmp_req=2 ttl=64 time=0.383 ms\n\n--- 192.168.12.2 ping statistics ---\n2 packets transmitted, 2 received, 0% packet loss, time 1000ms\nrtt min/avg/max/mdev = 0.383/0.400/0.418/0.026 ms"
    ],
    "stdout_lines": [
        [
            "PING 192.168.11.2 (192.168.11.2) 56(84) bytes of data.",
            "64 bytes from 192.168.11.2: icmp_req=1 ttl=64 time=0.383 ms",
            "64 bytes from 192.168.11.2: icmp_req=2 ttl=64 time=0.285 ms",
            "",
            "--- 192.168.11.2 ping statistics ---",
            "2 packets transmitted, 2 received, 0% packet loss, time 999ms",
            "rtt min/avg/max/mdev = 0.285/0.334/0.383/0.049 ms"
        ],
        [
            "PING 192.168.12.2 (192.168.12.2) 56(84) bytes of data.",
            "64 bytes from 192.168.12.2: icmp_req=1 ttl=64 time=0.418 ms",
            "64 bytes from 192.168.12.2: icmp_req=2 ttl=64 time=0.383 ms",
            "",
            "--- 192.168.12.2 ping statistics ---",
            "2 packets transmitted, 2 received, 0% packet loss, time 1000ms",
            "rtt min/avg/max/mdev = 0.383/0.400/0.418/0.026 ms"
        ]
    ]
}

TASK [Test reachability spine02 to leafs] *******************************************************************************************
task path: /home/vagrant/ansible-networking-vagrant-demo/test/check.yml:13
skipping: [leaf01] => {
    "changed": false,
    "skip_reason": "Conditional result was False",
    "skipped": true
}
skipping: [leaf02] => {
    "changed": false,
    "skip_reason": "Conditional result was False",
    "skipped": true
}
skipping: [spine01] => {
    "changed": false,
    "skip_reason": "Conditional result was False",
    "skipped": true
}
<172.16.10.14> attempting to start connection
<172.16.10.14> using connection plugin network_cli
<172.16.10.14> local domain socket does not exist, starting it
<172.16.10.14> control socket path is /home/vagrant/.ansible/pc/f972c41acf
<172.16.10.14> connection to remote device started successfully
<172.16.10.14> local domain socket listeners started successfully
<172.16.10.14>
<172.16.10.14> local domain socket path is /home/vagrant/.ansible/pc/f972c41acf
Using module file /usr/lib/python2.7/site-packages/ansible/modules/network/vyos/vyos_command.py
<172.16.10.14> ESTABLISH LOCAL CONNECTION FOR USER: vagrant
<172.16.10.14> EXEC /bin/sh -c 'echo ~ && sleep 0'
<172.16.10.14> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/vagrant/.ansible/tmp/ansible-tmp-1518108155.49-268149089724169 `" && echo ansible-tmp-1518108155.49-268149089724169="` echo /home/vagrant/.ansible/tmp/ansible-tmp-1518108155.49-268149089724169 `" ) && sleep 0'
<172.16.10.14> PUT /home/vagrant/.ansible/tmp/ansible-local-8825rmnY9F/tmp95ato7 TO /home/vagrant/.ansible/tmp/ansible-tmp-1518108155.49-268149089724169/vyos_command.py
<172.16.10.14> EXEC /bin/sh -c 'chmod u+x /home/vagrant/.ansible/tmp/ansible-tmp-1518108155.49-268149089724169/ /home/vagrant/.ansible/tmp/ansible-tmp-1518108155.49-268149089724169/vyos_command.py && sleep 0'
<172.16.10.14> EXEC /bin/sh -c '/usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1518108155.49-268149089724169/vyos_command.py && sleep 0'
The full traceback is:
Traceback (most recent call last):
  File "/tmp/ansible_TSqk5J/ansible_module_vyos_command.py", line 241, in <module>
    main()
  File "/tmp/ansible_TSqk5J/ansible_module_vyos_command.py", line 211, in main
    responses = run_commands(module, commands)
  File "/tmp/ansible_TSqk5J/ansible_modlib.zip/ansible/module_utils/network/vyos/vyos.py", line 117, in run_commands
  File "/tmp/ansible_TSqk5J/ansible_modlib.zip/ansible/module_utils/connection.py", line 140, in __rpc__
  File "/tmp/ansible_TSqk5J/ansible_modlib.zip/ansible/module_utils/connection.py", line 115, in _exec_jsonrpc
ansible.module_utils.connection.ConnectionError: socket_path does not exist or cannot be found

fatal: [spine02]: FAILED! => {
    "changed": false,
    "failed": true,
    "module_stderr": "Traceback (most recent call last):\n  File \"/tmp/ansible_TSqk5J/ansible_module_vyos_command.py\", line 241, in <module>\n    main()\n  File \"/tmp/ansible_TSqk5J/ansible_module_vyos_command.py\", line 211, in main\n    responses = run_commands(module, commands)\n  File \"/tmp/ansible_TSqk5J/ansible_modlib.zip/ansible/module_utils/network/vyos/vyos.py\", line 117, in run_commands\n  File \"/tmp/ansible_TSqk5J/ansible_modlib.zip/ansible/module_utils/connection.py\", line 140, in __rpc__\n  File \"/tmp/ansible_TSqk5J/ansible_modlib.zip/ansible/module_utils/connection.py\", line 115, in _exec_jsonrpc\nansible.module_utils.connection.ConnectionError: socket_path does not exist or cannot be found\n",
    "module_stdout": "",
    "msg": "MODULE FAILURE",
    "rc": 1
}
```

Also check out the demo of this whole diagram and setup https://github.com/IPvSean/ansible-networking-vagrant-demo

### PR DESCRIPTION
**Pull request by ganeshrn**
##### SUMMARY

Fix #35914

Command timeout and connection timeout error messages
are displayed in the log file instead of on the console.
Update the same in the troubleshooting guide.

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

rst/networking_guide/network_debug_troubleshooting.rst

##### ANSIBLE VERSION

```
2.5
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: docs/docsite/rst/networking_guide/network_debug_troubleshooting.rst [hunk 1]
```diff
@@ -138,6 +138,64 @@ Then review the log file and find the relevant error message in the rest of this
 
 .. For details on other ways to authenticate, see LINKTOAUTHHOWTODOCS.
 
+.. _socket_path_issue:
+
+Category "socket_path issue"
+========================================================
```
Comment (gundalow) [inline docs/docsite/rst/networking_guide/network_debug_troubleshooting.rst]: Only need `=` as long as the line above

File: docs/docsite/rst/networking_guide/network_debug_troubleshooting.rst [hunk 2]
```diff
@@ -138,6 +138,64 @@ Then review the log file and find the relevant error message in the rest of this
...
+
+**Platforms:** Any
+
+The ``socket_path does not exist or cannot be found``  and ``unable to connect to socket`` message is new in Ansible 2.5. This message means that
```
Comment (gundalow) [inline docs/docsite/rst/networking_guide/network_debug_troubleshooting.rst]: "message is new" -> "messages are new"
"This message means" -> "These messages means"

File: docs/docsite/rst/networking_guide/network_debug_troubleshooting.rst [hunk 3]
```diff
@@ -138,6 +138,64 @@ Then review the log file and find the relevant error message in the rest of this
...
+       "failed": true,
+       "module_stderr": "Traceback (most recent call last):\n  File \"/tmp/ansible_TSqk5J/ansible_modlib.zip/ansible/module_utils/connection.py\", line 123, in _exec_jsonrpc\nansible.module_utils.connection.ConnectionError: unable to connect to socket\n",
+       "module_stdout": "",
+       "msg": "MODULE FAILURE",
+       "rc": 1
+   }
+
+Suggestions to resolve:
+
+Follow the steps detailed in enable_network_logging_.
```
Comment (gundalow) [inline docs/docsite/rst/networking_guide/network_debug_troubleshooting.rst]: May be GitHub formatting issue, though this doesn't look right.
I think you want
```
 :ref:`enable network logging <enable_network_logging>`
```

File: docs/docsite/rst/networking_guide/network_debug_troubleshooting.rst [hunk 4]
```diff
@@ -138,6 +138,64 @@ Then review the log file and find the relevant error message in the rest of this
...
+
+If the identified the error message from the log file is
```
Comment (gundalow) [inline docs/docsite/rst/networking_guide/network_debug_troubleshooting.rst]: "log file is:"

File: docs/docsite/rst/networking_guide/network_debug_troubleshooting.rst [hunk 5]
```diff
@@ -138,6 +138,64 @@ Then review the log file and find the relevant error message in the rest of this
...
+
+For example:
```
Comment (gundalow) [inline docs/docsite/rst/networking_guide/network_debug_troubleshooting.rst]: Delete this and a blank line

File: docs/docsite/rst/networking_guide/network_debug_troubleshooting.rst [hunk 6]
```diff
@@ -138,6 +138,64 @@ Then review the log file and find the relevant error message in the rest of this
...
+
+   2017-04-04 12:19:05,670 p=18591 u=fred |  command timeout triggered, timeout value is 10 secs
+
+or
+
+.. code-block:: yaml
+
+   2017-04-04 12:19:05,670 p=18591 u=fred |  persistent connection idle timeout triggered, timeout value is 30 secs
+
+Follow the steps detailed in timeout_issues_.
```
Comment (gundalow) [inline docs/docsite/rst/networking_guide/network_debug_troubleshooting.rst]: Same as above think you need `:ref`

**Review by gundalow — CHANGES_REQUESTED** [review]
Thanks, just a few minor comments

**Comment by gundalow** [comment]
Thanks for this. Will need approval from both @acozine and @dharmabumstead 

Once they are happy this will need to go into `devel` & `stable-2.5`

**Comment by ganeshrn** [comment]
@gundalow Fixed comments, thank you for review.

ready_for_review

**Review by dharmabumstead — APPROVED** [review]
OK with some edits.

**Review by gundalow — APPROVED** [review]

**Review by rcarrillocruz — APPROVED** [review]

**Review by trishnaguha — APPROVED** [review]

**Review by acozine — APPROVED** [review]
Good info to include with the release. Thanks @ganeshrn.

**Comment by ganeshrn** [comment]
@gundalow @acozine @dharmabumstead @trishnaguha @rcarrillocruz Thank you for review.

**Comment by ganeshrn** [comment]
Cherry-pick to stable-2.5 51673f431f7f8c6ae4dcbdfac97e7d00b4203e46

**Pull request merged**